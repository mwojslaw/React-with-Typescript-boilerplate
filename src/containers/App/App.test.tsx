// import * as React from "react";
// import * as renderer from "react-test-renderer";
// import { configure } from "enzyme";
// import * as Adapter from "enzyme-adapter-react-16";
// import { shallow } from "enzyme"
// import { NotConnected } from "./";
// import Link from "./../Link";

// configure({ adapter: new Adapter() });

// test("App should render", () => {
//     const helloMessage = "Hello world";
//     const component = renderer.create(
//         <NotConnected helloMessage={helloMessage} usedTechnologies={[]} />
//     );

//     const tree = component.toJSON();
//     expect(tree).toMatchSnapshot();
// });

// test("App should render <div> as first child", () => {
//     const component = shallow(
//         <NotConnected helloMessage={''} usedTechnologies={[]} />
//     );

//     expect(component.first().is("div")).toBe(true);
// })

// test("App should render <h1> with passed message", () => {
//     const helloMessage = "Hello world";
//     const component = shallow(
//         <NotConnected helloMessage={helloMessage} usedTechnologies={[]} />
//     );

//     expect(component.find("h1").text()).toEqual(helloMessage);
// });

// test("App should render as meny <Link> as passed used technologies", () => {
//     const usedTechnologies = [{
//         href: '',
//         name: 'React'
//     }];

//     const component = shallow(
//         <NotConnected usedTechnologies={usedTechnologies} helloMessage='' />
//     );

//     expect(component.find(Link).length).toEqual(usedTechnologies.length);
// });