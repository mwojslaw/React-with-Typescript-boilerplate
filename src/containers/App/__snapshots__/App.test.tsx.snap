// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render 1`] = `
<div
  className="App"
>
  <h1>
    Hello world
  </h1>
  <ul />
</div>
`;
